universe/marathon.json.mustache

```diff
@@ -50,15 +50,26 @@
               "mode": "RW"
             }
           {{/storage.host-volume}}
-          {{^storage.host-volume}}
+          {{^storage.external-persistent-volume-name}}
             {
               "persistent": {
-                "size": {{storage.persistent-volume-size}}
+                "size": {{storage.local-persistent-volume-size}}
               },
               "mode": "RW",
               "containerPath": "jenkins_home"
```

Comment:
Wait. Is this the right container path? Host volume is at a different place?

Reply:
External/local persistent volumes have to be relative to the sandbox, so there's a JENKINS_HOME variable that changes it to be sandbox/jenkins_home